src/server/db/schema.ts

```diff
@@ -36,6 +36,17 @@ export const usersRelations = relations(users, ({ one }) => ({
 export const userSchema = createInsertSchema(users);
 export type User = z.infer<typeof userSchema>;
 
+export const userAchievements = bbPgTable("user_achievement", {
+  userId: uuid("user_id")
+    .notNull()
+    .references(() => users.id, { onDelete: "cascade" }),
+  achievementId: varchar("achievement_id", { length: 32 }).notNull(),
+  achievedAt: timestamp("achieved_at").notNull(),
+});
+
+export const userAchievementsSchema = createInsertSchema(userAchievements);
```

Comment:
You are using it as a `createSelectSchema` later though from what I see

Reply:
I am also using this to insert data, What is the difference?

Reply:
https://orm.drizzle.team/docs/zod